app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -117,9 +126,16 @@ You can also learn English with a private provider. Well respected English langu
 * [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl)
 * [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)
 
+### Qualified teacher status
 
+[Qualified teacher status (QTS)]((https://www.gov.uk/guidance/qualified-teacher-status-qts)) is the professional status teachers in England gain at the end of their teacher training. Schools use QTS as a measure of teaching skills and experience, so you will find it much easier to get a teaching job in England if you have QTS.
 
-[Find out more about teaching in England if you trained overseas](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
+### Apply for QTS in England
+
+From 1 February 2023, qualified teachers from Ukraine who [meet the criteria for QTS](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers) will be able to [apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start) without paying a fee or undergoing more training. 
+
+### Assessment only QTS
+If you find you don't meet the eligibility criteria to apply for QTS, you could consider assessment only QTS. This means you can work in an English school and, alongside your job, apply to an [English teacher training provider](/assessment-only-providers) who will assess if you meet the standards for English QTS. DfE has made funding available for candidates who apply for assessment only courses. Check assessment only training providers for more information about eligibility and funding.
```

Comment:
```suggestion
If you find you do not meet the eligibility criteria to apply for QTS, you could consider assessment only QTS. This means you can work in an English school and, alongside your job, apply to an English teacher training provider who will assess if you meet the standards for English QTS.

DfE has made funding available for candidates who apply for assessment only courses. Check with [assessment only training providers]((/assessment-only-providers) for more information about eligibility and funding.
```
Thought it made sense to break up this big block of text, and also move the link. It didn't feel like the right link text to take them to that page for me? Hope this makes sense